src/ns_tracker/nsdeps.clj

```diff
@@ -21,8 +22,24 @@
 	     (contains? #{:use :require} (first form)))
     (apply union (map #(deps-from-libspec nil %) (rest form)))))
 
+(defn- find-resource [path dirs]
+  (->> (map #(File. ^File % ^String path) dirs)
+       (filter #(.isFile ^File %))
+       first))
+
+(defn- deps-from-ns-metadata [decl dirs]
+  (if-let [deps (:resource-deps (meta (second decl)))]
```

Comment:
Let's call the metadata key `:ns-tracker/extra-deps`.

Reply:
The name "extra-deps" doesn't describe that what kind of dependencies they are - namespace dependencies could also be called "extra deps". But these dependencies are specifically classpath resource dependencies, so I would prefer calling them resource deps or file deps.

It's likely that this same feature will need to be implemented also in clojure.tools.namespace, because not every tool uses ns-tracker. A standardized metadata key could be used by all libraries, though probably it's too early to standardize yet. What do you think about having to duplicate the configuration? For example `{:ns-tracker/resource-deps ["file.sql"], :clojure.toos.namespace/resource-deps ["file.sql"]}` It would be similar to the use of [CSS browser prefixes](https://www.lifewire.com/css-vendor-prefixes-3466867) until all vendors settle on a standard.

Reply:
If they're only resources, then `:ns-tracker/resource-deps` seems fine.